ui/app/models/cluster.js

```diff
@@ -49,47 +49,17 @@ export default DS.Model.extend({
 
   anyReplicationEnabled: or('{dr,performance}.replicationEnabled'),
 
-  stateDisplay(state) {
-    if (!state) {
-      return null;
+  modeState: computed(
+    'dr.mode',
+    'performance.mode',
+    'replicationMode',
+    'dr.state',
+    'performance.state',
+    function() {
+      const mode = this.replicationMode;
+      return this.get(`${mode}.state`);
     }
-    const defaultDisp = 'Synced';
-    const displays = {
```

Comment:
you can see these values are now part of the `CLUSTER_STATES` map here: https://github.com/hashicorp/vault/pull/8787/files#diff-b8f0db91f050a7646bbb0d17521230a2R20